app/store/attachment.go

```diff
@@ -36,6 +37,7 @@ func SaveAttachment(a *textsecure.Attachment) (Attachment, error) {
 	// 	ext = strings.Replace(a.MimeType, "video/", ".", 1)
 	// }
 	fileName := a.FileName
+	fileName = strings.Replace(fileName, "/", "-", -1)
```

Comment:
@nalysius thanks for your fix, can you check, why deepsource is complaining in this report? https://deepsource.io/gh/nanu-c/axolotl/run/af86c433-edc5-4277-be10-e48aafc0917a/go/CRT-A0010

Reply:
It prefers the use of ReplaceAll() over Replace() with -1 as the last parameter, it passes now.